packages-exp/auth-exp/test/api/helper.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Auth } from '../../src/model/auth';
+import { Endpoint } from '../../src/api';
+import { mock, Route } from '../mock_fetch';
+
+const TEST_HOST = 'localhost';
+const TEST_SCHEME = 'mock';
+const TEST_KEY = 'test-api-key';
+
+export const mockAuth: Auth = {
```

Comment:
Eventually I think we will want to have this in its own test helper so that we can configure different properties on the auth object. Up to you if you want to do that now or not

Reply:
let's consolidate after we merge https://github.com/firebase/firebase-js-sdk/pull/2896